CMakeLists.txt

```diff
@@ -111,11 +111,6 @@ if (NOT MINGW AND NOT UNIX)
     message(FATAL_ERROR "Unsupported operating system / toolset")
 endif()
 
-if (CMAKE_Fortran_COMPILER_ID MATCHES "Clang")
```

Comment:
Is seomeone setting `CMAKE_Fortran_COMPILER_ID` to `${CMAKE_C_COMPILER_ID}` somewhere? In new CMakes it detects Fortran correctly. Not 100% sure what the status of 3.1 is, however.

Reply:
This check didn't make sense in the first place, that's why I removed it. Just some cleanup.

Reply:
It's entirely possible that 3.1 doesn't support `Fortran_COMPILER_ID`, btw. I don't know if you've tested everything with an ancient 3.1 CMake, but it may be worth your while to bump the minimum required version up at least a little bit, say 3.4. But up to you.